imports/plugins/included/product-variant/components/customer/productGridItem.js

```diff
@@ -14,7 +14,7 @@ class ProductGridItem extends Component {
       isLowQuantity: PropTypes.bool,
       isSoldOut: PropTypes.bool,
       media: PropTypes.arrayOf(PropTypes.object),
-      pricing: PropTypes.object,
+      pricing: PropTypes.array,
```

Comment:
Since you are relying on the shape of these, do `PropTypes.arrayOf(PropTypes.shape({ ... }))` with the correct props inside shape.